src/main/java/io/reactivex/Single.java

```diff
@@ -874,7 +874,7 @@
      * @return the new Single instance
      * @since 2.0
      */
-    public static <T, R> Single<R> zip(final Iterable<? extends SingleSource<? extends T>> sources, Function<? super Object[], ? extends R> zipper) {
+    public static <T, R> Single<? extends R> zip(final Iterable<? extends SingleSource<? extends T>> sources, Function<? super Object[], ? extends R> zipper) {
```

Comment:
Not quite the latest. I'm running Android Studio beta2. So should I change this PR to cast the value and suppress the warning?